SimCalorimetry/HcalTrigPrimProducers/plugins/HcalTrigPrimDigiProducer.cc

```diff
@@ -192,20 +192,20 @@ void HcalTrigPrimDigiProducer::beginRun(const edm::Run& run, const edm::EventSet
       if (!overrideDBweightsAndFilterHB_) {
         if (fixedPointWeight != -1.0) {
```

Comment:
fixedPointWeight is an integer, please don't ask it being different from a float!

Analogously, please modify the initialization with a "-1" integer value in
https://github.com/cms-sw/cmssw/blob/7575645d64e16d81845a3b0b1e583a666e3f5cf2/SimCalorimetry/HcalTrigPrimProducers/plugins/HcalTrigPrimDigiProducer.cc#L178